app/src/main/java/com/example/myapplication/LoginFragment.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.myapplication
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.fragment.app.Fragment
+
+class LoginFragment(val credentialsManager : CredentialsManager) : Fragment() {
+
```

Comment:
Sometimes you put an empty line here, at the beginning of the class - sometimes not
Stay consistent